src/Compilers/CSharp/Test/Emit/PDB/PDBAsyncTests.cs

```diff
@@ -2016,11 +2016,13 @@ public partial class C
 
             AssertEx.AssertEqualToleratingWhitespaceDifferences(@"
 MethodDebugInformation (index: 0x31, size: 20): 
-==================================================
-1: nil
-2: nil
-3: nil
-4:
+================================================
```

Comment:
This change (and similar changes in other tests) were required due to the mdv version update.